test/constraints-sram.sdc

```diff
@@ -1,4 +1,4 @@
-source util.tcl
```

Comment:
move util.tcl into the test/ folder. It is not part of bazel-orfs, but part of the design being built.

Reply:
Done